### ISSUE TITLE
Update the computing likelihoods page

### ISSUE THREAD

**Issue by jeromekelleher** [tskit-dev/msprime#1423]
The current [computing likelihoods](https://tskit.dev/msprime/docs/latest/likelihoods.html) page needs a little work

1. A little bit of background as to what we're doing an why we'd want to do it.
1. Change the text to refer to sim_ancestry rather than simulate
2. Add a Quick Reference section
3. Give a little bit of explanation about why ``discrete_genome`` must be False
4. Give a little bit more explanation as to what's happening in the examples, and maybe do them one-by-one (or maybe make a dict like ``{rho: msprime.log_arg_likelihood(ts, recombination_rate=rho, Ne=1) for rho in [0, 0.1, 1, 10]}``) rather than just doing a few prints. We could even consider plotting these values on a graph, or something?

Basically, it's a bit terse and technical at the moment and could do with a little more explanation. Basically, the docstrings are for terse, precise technical information and the explanation sections are for illustrating how things work to users at all levels.

See also #1411.

@JereKoskela, you're best positioned to pick this one up; any chance you could fit it in? The new [documentation docs](https://tskit.dev/msprime/docs/latest/development.html#documentation) should help orient you with the new way things are done.

**Comment by JereKoskela** [comment]
Yeah, I should be able to redraft this without much difficulty. Leave it with me.

**Comment by JereKoskela** [comment]
I've run into a naive question @jeromekelleher: What do you mean by a Quick Reference section? The developer docs talk about the API reference with technical specifications, and both likelihoods are already in there, but it sounds like you mean something different.

**Comment by jeromekelleher** [comment]
I meant a section like [this one](https://tskit.dev/msprime/docs/latest/ancestry.html#quick-reference) in the ancestry page. The material is probably already there, it just literally needs a section heading I'd imagine.
